server/src/main/java/org/elasticsearch/index/translog/Checkpoint.java

```diff
@@ -164,7 +210,7 @@ public static void write(ChannelFactory factory, Path checkpointFile, Checkpoint
         };
         final String resourceDesc = "checkpoint(path=\"" + checkpointFile + "\", gen=" + checkpoint + ")";
         try (OutputStreamIndexOutput indexOutput =
-                 new OutputStreamIndexOutput(resourceDesc, checkpointFile.toString(), byteOutputStream, FILE_SIZE)) {
+                 new OutputStreamIndexOutput(resourceDesc, checkpointFile.toString(), byteOutputStream, V2_FILE_SIZE)) {
```

Comment:
Wait, don't you want `FILE_SIZE` here (or `V3_FILE_SIZE` with my naming suggestion)